### PR TITLE
Clean up and refactor digital asset template

### PR DESCRIPTION
**Pull request by 0xmaayan**
Clean up and refactor digital asset code, folder structure, file names

### CODE REVIEW

File: templates/digital-asset-template/frontend/config.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import Placeholder3 from "@/assets/placeholders/bear-3.png";
 
 export const config: Config = {
   // TODO: Fill in your collection id
-  collection_id: "",
+  collection_id: "0xb55b45a33f3dc1c6a889cf3bd5b9bed1dac72aeb829fc613f9458b59c3d5eb13",
```
Comment (kaw2k) [inline templates/digital-asset-template/frontend/config.ts:7]: Remove

File: templates/digital-asset-template/frontend/pages/Mint/components/HeroSection.tsx [hunk 2]
```diff
@@ -1,30 +1,36 @@
...
-import { formatDate } from "@/utils/formatDate";
 import { Progress } from "@/components/ui/progress";
 
+import { NETWORK } from "@/constants";
+
+import { config } from "@/config";
+
+import { mintNFT } from "@/entry-functions/mint_nft";
+
+import { Socials } from "./Socials";
```
Comment (kaw2k) [inline templates/digital-asset-template/frontend/pages/Mint/components/HeroSection.tsx:28]: nit (feel free to ignore): All these extra spaces can be removed. Also, do we want to stick with the `@/` convention for `Socials`?

File: templates/digital-asset-template/frontend/utils/Irys.ts [hunk 3]
```diff
@@ -26,16 +26,10 @@ export const checkIfFund = async (aptosWallet: WalletContextState, files: File[]
...
-  const currentAccountBalance = await aptosClient().view<[number]>({
-    payload: {
-      function: "0x1::coin::balance",
-      typeArguments: ["0x1::aptos_coin::AptosCoin"],
-      functionArguments: [currentAccountAddress],
-    },
-  });
-
+  const currentAccountBalance = await accountAPTBalance({ accountAddress: currentAccountAddress });
+  console.log("currentAccountBalance", currentAccountBalance);
```
Comment (kaw2k) [inline templates/digital-asset-template/frontend/utils/Irys.ts:30]: Remove

**Review by kaw2k — COMMENTED** [review]
Did you run all this through `npm run fmt`? I feel like some of these lines are longer than 120.

File: templates/digital-asset-template/frontend/pages/Mint/components/HeroSection.tsx (hunk 2 above)
Comment (0xmaayan) [reply templates/digital-asset-template/frontend/pages/Mint/components/HeroSection.tsx:28]: I wanted to make it more readable, probably want to add comments on what each file is

**Review by kaw2k — APPROVED** [review]

**Comment by 0xmaayan** [comment]
> Did you run all this through `npm run fmt`? I feel like some of these lines are longer than 120.

yes, it passes. we also have a fmt check on CI

**Pull request merged**